Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /nlisim
+COPY . /nlisim
+
+RUN apt update \
+	&& apt install -y git libgl1-mesa-glx \
```

Comment:
Why is `git` necessary?

Reply:
It seems like in order to build the image, it needs the .git folder and the git library. Otherwise, setuptools-scm will throw an exception:
```
    LookupError: setuptools-scm was unable to detect version for '/tmp/pip-req-build-9qw_9rzf'.
    
    Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.
    
    For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
```
This doesn't look right to me and the image should be built without any dependency on Git. Any idea?

Reply:
This is a requirement of setuptools-scm to automatically generate the version number for release on CI.  There is some discussion about this topic in https://github.com/pypa/setuptools_scm/issues/77.